src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/StorageAccount/SyncStorageAccount.cs

```diff
@@ -61,9 +68,15 @@ public sealed class SyncStorageAccount : AdminCmdlet
         /// </summary>
         [Parameter(Mandatory = false, Position = 8)]
         public string StorageAccountApiVersion { get; set; }
-        
+
+        /// <summary>
+        /// Specifies the Microsoft.Resource.Admin apiVersion
+        /// </summary>
+        [Parameter(Mandatory = false, Position = 9)]
```

Comment:
I will remove the position attribute for new parameters while leaving the ones already exist. Sounds reasonable?